sdk/servicebus/service-bus/src/connectionContext.ts

```diff
@@ -351,10 +356,10 @@ export namespace ConnectionContext {
       // The connection should always be brought back up if the sdk did not call connection.close()
       // and there was atleast one sender/receiver link on the connection before it went down.
       logger.verbose("[%s] state: %O", connectionContext.connectionId, state);
-      if (!state.wasConnectionCloseCalled && (state.numSenders || state.numReceivers)) {
+      if (!state.wasConnectionCloseCalled && state.numReceivers) {
         logger.verbose(
           "[%s] connection.close() was not called from the sdk and there were some " +
-            "senders and/or receivers. We should reconnect.",
+            "receivers. We should reconnect.",
```

Comment:
Probably worth printing out the number - might be useful to figure out how disastrous the reconnect was for the user.

Reply:
(above too, i think)

Reply:
Nice, can do that!